python/pyspark/ml/wrapper.py

```diff
@@ -263,7 +284,8 @@ def _fit_java(self, dataset):
 
     def _fit(self, dataset):
         java_model = self._fit_java(dataset)
-        return self._create_model(java_model)
+        model = self._create_model(java_model)
+        return self._copyValues(model)
```

Comment:
Here I think it is going to copy values from the estimator to the created model. So I think we assume that the params in estimator and model are the same?